components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultLogoutRequestHandler.java

```diff
@@ -256,6 +256,12 @@ public void handle(HttpServletRequest request, HttpServletResponse response, Aut
                 if (authenticatorConfig == null) {
                     authenticatorConfig = sequenceConfig.getAuthenticatedReqPathAuthenticator();
                 }
+                // If there is no authenticated authenticator, skip the step.
+                if (authenticatorConfig == null) {
```

Comment:
Introduce a unit test covering this case.

Reply:
fixed